Make constant_folding's _DEFAULT_SKIP_TARGETS public

ThomasJannaud opened the pull request:
Summary: This follows D73513516 and allows customers of constant_prop_pass to use previous defaults

Differential Revision: D74349918

JacobSzwejbka reviewed (approved):
looks a bit weird since both sets are the same here but I guess its fine

ThomasJannaud committed:
Make constant_folding's _DEFAULT_SKIP_TARGETS public (#10760)

Summary:

This follows D73513516 and allows customers of constant_prop_pass to use previous defaults
This goes with D74350331 and  D74280168

Reviewed By: JacobSzwejbka

Differential Revision: D74349918
